lib/driver/index.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+const moment = require('moment');
+const async = require('async');
+const Schema = mongoose.Schema;
+const driverSchema = new Schema({
+  email: String,
+  name: String,
+  city: String,
+  phoneNumber: String,
+  createdOn: {
+    type: Date,
+    default: Date.now,
+  },
+});
+const DriverModel = mongoose.model('Driver', driverSchema);
+
+class Driver extends DriverModel {
+  getCurrentDate() {
+    return moment().format();
+  }
+  create(cb) {
+    async.series([
```

Comment:
I usually use async.series when there are several async tasks that need to get executed in series, since you're only doing the this.save, I would remove the async.series